.github/workflows/ci.yml

```diff
@@ -24,30 +24,31 @@ jobs:
     name: test
     env:
       RUST_VERSION: ${{ matrix.rust }}
+      TARGET: ${{ matrix.target }}
     strategy:
       matrix:
-        crates:
-          - crossbeam
-          - crossbeam-channel
-          - crossbeam-deque
-          - crossbeam-epoch
-          - crossbeam-queue
-          - crossbeam-skiplist
-          - crossbeam-utils
-        rust:
-          - 1.36.0
-          - nightly
-        os:
-         - ubuntu-latest
-         - windows-latest
-    runs-on: ${{ matrix.os }}
+        include:
+          - rust: 1.36.0
+          - rust: 1.36.0
+            os: windows-latest
+          - rust: nightly
+          # TODO: https://github.com/crossbeam-rs/crossbeam/pull/518#issuecomment-633342606
+          # - rust: nightly
+          #   os: macos-latest
+          - rust: nightly
+            os: windows-latest
+          - rust: nightly
+            target: i686-unknown-linux-gnu
+          - rust: nightly
+            target: aarch64-unknown-linux-gnu
+    runs-on: ${{ matrix.os || 'ubuntu-latest' }}
```

Comment:
We're effectively marking `ubuntu-latest` as the "default" platform here. I prefer not to select one as the default for simplicity. What do you think?

Reply:
Indeed. Changed to always set `matrix.os`.